.github/workflows/main.yml

```diff
@@ -107,4 +98,4 @@ jobs:
             **/testdata/*.tar
             **/testdata/*.tar.gz
       - name: Tests
-        run: make integration
+        run: go test -race -tags integration ./...
```

Comment:
What is the idea here? Enable the cache?

Reply:
Yeah, to enable the cache. It'll get blown away on every toolchain or dependency upgrade.